pkg/aws_config_server/shared_variables_test.go

```diff
@@ -205,51 +206,66 @@ var alternatePolicyDocument = &AlternatePolicyDocument{
 
 var testRoles0 = []*iam.Role{
 	{
-		Arn:      aws.String("roleARN"),
+		Arn:      aws.String(BareRoleARN("roleARN").String()),
 		RoleName: aws.String("testRoles0"),
 	},
 }
 var testRoles1 = []*iam.Role{
 	{
-		Arn:      aws.String("roleARN0"),
+		Arn:      aws.String(BareRoleARN("roleARN0").String()),
```

Comment:
I'm curious about the purpose of the BareRoleARN function here. It appears to me that the BareRoleARN function gets passed a string, then returns a pointer to an arn object. Once we receive that return value, we simply convert it back into a string. I could be misreading this though as I don't read Go very well but would be interested to learn why it's necessary here.

Reply:
I'm lazy and didn't want to figure out how to write `arn:::::::roleARN0` so instead wrote a little function that would do it for me.